DOC-1082 add BYOC GCP europe-southwest1 (#225)
redpanda-data/cloud-docs: modules/get-started/pages/whats-new-cloud.adoc

This page lists new features added in Redpanda Cloud.

== March 2025

=== Support for additional regions

> For xref:reference:tiers/byoc-tiers.adoc#byoc-supported-regions[BYOC clusters], Redpanda added support for the GPC europe-southwest1 region (Madrid).

Reviewer: ```suggestion
xref:reference:tiers/byoc-tiers.adoc#byoc-supported-regions[BYOC clusters] on GCP now support the europe-southwest1 (Madrid) region .
```
